package.json

```diff
@@ -99,7 +99,7 @@
     "typescript": "^5.6.3"
   },
   "dependencies": {
-    "@gusto/embedded-api": "^0.5.4",
+    "@gusto/embedded-api": "^0.5.6",
```

Comment:
Necessary to bring in a fix for the invite signatory response code which was preventing the query from working before. See https://gustohq.slack.com/archives/C08HVS4MJQN/p1743004912630929